src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { Alert, Button, Level, LevelItem, Pagination } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody, sortable, ICell, IRow } from '@patternfly/react-table';
+import { usePaginationState, useSortState } from '@app/common/hooks';
+import { useSelectionState } from '@konveyor/lib-ui';
+import { IVMwareProvider, IHost } from '@app/queries/types';
+import { formatHostNetwork } from './helpers';
+import SelectNetworkModal from './SelectNetworkModal';
+import { useHostsQuery } from '@app/queries';
+import LoadingEmptyState from '@app/common/components/LoadingEmptyState';
+
+interface IVMwareProviderHostsTableProps {
+  providerId: string;
+}
+
+const VMwareProviderHostsTable: React.FunctionComponent<IVMwareProviderHostsTableProps> = ({
+  providerId,
+}: IVMwareProviderHostsTableProps) => {
+  const hostsQuery = useHostsQuery(providerId);
+
+  const hosts: IHost[] = !hostsQuery.data ? [] : hostsQuery.data;
+
+  const columns: ICell[] = [
+    { title: 'Name', transforms: [sortable] },
+    { title: 'Network for migration data transfer', transforms: [sortable] },
+    { title: 'Bandwidth', transforms: [sortable] },
+    { title: 'MTU', transforms: [sortable] },
+  ];
+
+  const getSortValues = (host: IHost) => {
+    const { name, network, bandwidth, mtu } = host;
+    // First cell is the generated checkbox from using onSelect.
+    return ['', name, 'network', 'bandwidth', 'mtu'];
+    //TODO update when api data is available
+    // return ['', name, formatHostNetwork(network), bandwidth, mtu];
+  };
+  const { sortBy, onSort, sortedItems } = useSortState(hosts, getSortValues);
+  const { currentPageItems, setPageNumber, paginationProps } = usePaginationState(sortedItems, 10);
+  const { selectedItems, toggleItemSelected, selectAll } = useSelectionState<IHost>({
+    items: sortedItems,
+  });
+
+  const rows: IRow[] = sortedItems.map((host: IHost) => {
+    const { name } = host;
+    return {
+      meta: { host },
+      selected: selectedItems.includes(host),
+      cells: [name, 'network N/A', 'bandwidth N/A', 'mtu N/A'],
```

Comment:
Yeah, we need to get in touch with Jeff about how to get this data for hosts. Opened an issue: https://github.com/konveyor/virt-ui/issues/85